lib/tests/streamlit/elements/image_test.py

```diff
@@ -75,10 +74,7 @@ def create_image(size, format="RGB", add_alpha=True):
         )
         image.putalpha(alpha)
 
-    if format == "BGR":
```

Comment:
Removing the `BGR` test cases since they require an external dependency, which is not compatible with numpy 2.x, and they are not really adding a lot of additional test coverage. We aren't doing anything special to support the BGR color space, and I also think its a very niche thing.

Reply:
Do you know why this test was added originally? Looking at it indeed does not make it obvious why testing some image library is part of the Streamlit test suite and I agree with your conclusion to remove it